File: app/src/main/java/com/asianliftbd/staff/Start.java
package com.asianliftbd.staff;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.os.SystemClock;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;
import com.google.firebase.auth.FirebaseAuth;
import com.google.firebase.auth.FirebaseUser;

public class Start extends AppCompatActivity {
    private long C;
    private FirebaseUser user = FirebaseAuth.getInstance().getCurrentUser();
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out);
        setContentView(R.layout.start);
    }
    public void Login(View v) {
        if (SystemClock.elapsedRealtime()-C<1000){return;}
        C=SystemClock.elapsedRealtime();
        startActivity(new Intent(Start.this, Login.class));
    }
    public void Terms(View v){
        if (SystemClock.elapsedRealtime()-C<1000){return;}
        C=SystemClock.elapsedRealtime();
        String url="https://asianliftbd.com/terms-of-use";
        Intent i=new Intent(Intent.ACTION_VIEW);
        i.setData(Uri.parse(url));
        startActivity(i);
    }
    public void Right(View v) {
        if (SystemClock.elapsedRealtime()-C<1000){return;}
        C=SystemClock.elapsedRealtime();
        String url = "https://asianliftbd.com";
        Intent i = new Intent(Intent.ACTION_VIEW);
        i.setData(Uri.parse(url));
        startActivity(i);
    }
    @Override
    public void onStart() {
        super.onStart();
        if (user != null) {
            startActivity(new Intent(Start.this, Login.class));
            finish();
        }
    }
}

File: app/src/main/java/com/asianliftbd/staff/activity/Start.kt
package com.asianliftbd.staff.activity

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import com.asianliftbd.staff.R
import com.asianliftbd.staff.Utils
import com.asianliftbd.staff.databinding.ActivityStartBinding
import com.google.firebase.Firebase
import com.google.firebase.auth.FirebaseAuth
import com.google.firebase.initialize

class Start : AppCompatActivity() {
    private lateinit var binding: ActivityStartBinding
    private val user = FirebaseAuth.getInstance().currentUser

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        binding = ActivityStartBinding.inflate(layoutInflater)
        setContentView(binding.root)
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }

        Firebase.initialize(context = this)

        binding.loginButton.setOnClickListener {
            Utils.singleClick()
            startActivity(Intent(this, Login::class.java))
        }

        binding.websiteButton.setOnClickListener {
            Utils.singleClick()
            val i = Intent(Intent.ACTION_VIEW)
            i.data = Uri.parse("https://asianliftbd.com")
            startActivity(i)
        }

        binding.termsOfUse.setOnClickListener {
            Utils.singleClick()
            val i = Intent(Intent.ACTION_VIEW)
            i.data = Uri.parse("https://asianliftbd.com/terms-of-use")
            startActivity(i)
        }
    }

    override fun onStart() {
        super.onStart()
        if (user != null) {
            startActivity(Intent(this, Dashboard::class.java))
            finish()
        }
    }
}